<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AdminUI</class>
 <widget class="QMainWindow" name="AdminUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Administratorius</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="addUserButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>120</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>Pridėti vartotoją</string>
    </property>
   </widget>
   <widget class="QPushButton" name="deleteUserButton">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>10</y>
      <width>120</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>Ištrinti vartotoją</string>
    </property>
   </widget>
   <widget class="QPushButton" name="createGroupButton">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>10</y>
      <width>120</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>Sukurti grupę</string>
    </property>
   </widget>
   <widget class="QPushButton" name="assignGroupButton">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>10</y>
      <width>120</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>Priskirti grupę
 studentui</string>
    </property>
   </widget>
   <widget class="QPushButton" name="deleteGroupButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>120</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>Pašalinti grupę</string>
    </property>
   </widget>
   <widget class="QPushButton" name="addSubjectButton">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>50</y>
      <width>120</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>Sukurti dėstomą
dalyka</string>
    </property>
   </widget>
   <widget class="QPushButton" name="assignSubjectButton">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>50</y>
      <width>120</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>Priskirti dėstomą
dalyką dėstytojui</string>
    </property>
   </widget>
   <widget class="QPushButton" name="deleteSubjectButton">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>50</y>
      <width>120</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>Pašalinti dėstomą
 dalyką</string>
    </property>
   </widget>
   <widget class="QPushButton" name="logOffButton">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>10</y>
      <width>71</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>Atsijungti</string>
    </property>
   </widget>
   <widget class="QPushButton" name="viewFileButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>120</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>Peržiūra</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
